platforms/documentation/docs/src/docs/userguide/releases/upgrading/upgrading_version_8.adoc

```diff
@@ -75,6 +75,11 @@ This should not impact catalogs that use the <<platforms.adoc#sub::toml-dependen
 
 === Deprecations
 
+==== Deprecated registration of plugin conventions
+
+In addition to accessing plugin conventions (which emitted warnings since Gradle 8.2), registering plugin conventions will now also emit deprecation warnings.
```

Comment:
```suggestion
Using plugin conventions has been emitting warnings since Gradle 8.2. 
Now, registering plugin conventions will also trigger deprecation warnings.
```